client/src/components/App/Content/UsersManagement/UsersManagement.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
Isn’t this component a bit empty? Are you going to add anything else besides the table?

Reply:
Yes you right, Naaman already told me about it yesterday

Reply:
fixed